packages/markdown/src/index.ts

```diff
@@ -0,0 +1,317 @@
+import type { FileType, Service, FileChangeType } from '@volar/language-service';
+import MarkdownIt from 'markdown-it';
```

Comment:
I think it’s best to not rely on `esModuleInterop`. It generates some interop clutter and, if we’re not careful, it can cause TypeScript to emit invalid type definitions.

Reply:
`verbatimModuleSyntax` is a great tip! Therefore I'm now thinking of migrating to ESM for Volar.js.